### PR TITLE
Add SmartHab documentation page

### PR DESCRIPTION
**Pull request by outadoc**
**Description:**
This PR adds documentation for the SmartHab integration support in Home Assistant.

SmartHab is a French company that provides a proprietary app-based home automation platform in some new buildings, including mine. You can learn a bit more on their website: http://smarthab.fr 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21225

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/smarthab.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  might want to create a secondary user in the app's settings and grant it
+  access to your home. You can then configure the component using this account's
+  credentials.
+</p>
+
+```yaml
+# Example configuration.yaml entry
+smarthab:
+  email: john.doe@example.com
+  password: '123456'
```
Comment (klaasnicolaas) [inline source/_components/smarthab.markdown]:
```suggestion
  password: PASSWORD
```

File: source/_components/smarthab.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  To prevent being automatically logged out of your SmartHab mobile app, you
+  might want to create a secondary user in the app's settings and grant it
+  access to your home. You can then configure the component using this account's
+  credentials.
+</p>
+
+```yaml
+# Example configuration.yaml entry
+smarthab:
+  email: john.doe@example.com
```
Comment (klaasnicolaas) [inline source/_components/smarthab.markdown]:
```suggestion
  email: EMAIL_ADDRESS
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by outadoc** [comment]
Hi @klaasnicolaas, could you maybe take another look at the PRs? They should be complete now. Thanks :)

File: source/_components/smarthab.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+sharing: true
+footer: true
+logo: smarthab.png
+ha_release: 0.89
+ha_category:
+  - Hub
+  - Cover
+  - Light
+ha_iot_class: "Cloud Polling"
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/smarthab.markdown]: Please remove this

File: source/_components/smarthab.markdown [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+comments: false
+sharing: true
+footer: true
+logo: smarthab.png
+ha_release: 0.89
+ha_category:
+  - Hub
+  - Cover
+  - Light
+ha_iot_class: "Cloud Polling"
```
Comment (klaasnicolaas) [inline source/_components/smarthab.markdown]:
```suggestion
ha_iot_class: Cloud Polling
```

File: source/_components/smarthab.markdown [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+layout: page
+title: "SmartHab"
+description: "Instructions on how to integrate SmartHab devices into Home Assistant"
+date: 2019-02-13 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: smarthab.png
+ha_release: 0.89
```
Comment (klaasnicolaas) [inline source/_components/smarthab.markdown]:
```suggestion
ha_release: 0.92
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by outadoc** [comment]
@klaasnicolaas Thanks. Should be good now.

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
Yep it is now waiting on the parent PR 😉

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**